### ISSUE TITLE
On Permissions page, Screen Reader fails to announce “permissions is not available” text. 

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#229]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40128466

**Comment by AgneLukoseviciute** [comment]
possibly because of: https://github.com/microsoft/react-native-gallery/issues/224

**Comment by AgneLukoseviciute** [comment]
Testing on .70, this still reproes
